packages/core/src/telemetry/machine-id.ts

```diff
@@ -33,12 +34,13 @@ const calcMachineId = async () => {
 }
 
 export const memoizedMachineId = async (dataDir: string) => {
+  const dir = localFs(dataDir)
```

Comment:
why is `localFs` needed here?

Reply:
in container environments, datadir folder didn't exist